ext/opentelemetry-ext-requests/src/opentelemetry/ext/requests/__init__.py

```diff
@@ -157,7 +159,12 @@ def _http_status_to_canonical_code(code: int, allow_redirect: bool = True):
 
 class RequestsInstrumentor(BaseInstrumentor):
     def _instrument(self, **kwargs):
```

Comment:
Would you mind adding a docstring with the parameters this function accepts? 
For the reference, we don't know how to generate this documentation yet (https://github.com/open-telemetry/opentelemetry-python/issues/617), but the docstring is useful anyway.

Reply:
Added. LMK if the copy needs to be changed.